javascript/attributes.js

```diff
@@ -1,11 +1,17 @@
 import { defaultSchema } from './schema'
 import Debug from './debug'
 
-const multipleInstances = element =>
-  document.querySelectorAll(
-    `input[type="${element.type}"][name="${element.name}"]`
-  ).length > 1
-
+const multipleInstances = element => {
+  if (['checkbox', 'radio'].includes(element.type)) {
+    return (
+      document.querySelectorAll(
+        `input[type="${element.type}"][name="${element.name}"]`
+      ).length > 1
+    )
+  } else {
+    return false
```

Comment:
I suspect you can just get rid of the whole `else {}` and just return false here. False is a very reasonable default return for this function.

Reply:
Good suggestion, thanks.